book/coloring_and_theming.md

```diff
@@ -217,9 +213,33 @@ Example: `{ fg: "#ff0000" bg: "#0000ff" attr: b }`
 - background of blue in "#hex" format
 - attribute of bold abbreviated
 
-## Primitive Values
+### Closure
+
+Note: Closures are only executed for table output. They do not work in other contexts like for `shape_` configurations, when printing a value directly, or as a value in a list.
 
----
+For example:
+
+```nu
+$env.config.color_config.filesize = {|x| if $x == 0b { 'dark_gray' } else if $x < 1mb { 'cyan' } else { 'blue' } }
+$env.config.color_config.filesize = {|x| if $x == 0b { 'dark_gray' } else if $x < 1mb { 'cyan' } else { 'blue_bold' } }
```

Comment:
i'm not sure why there are two rows here each setting filesize but differently?

Reply:
oh I meant to include a bool format closure

Reply:
```suggestion
$env.config.color_config.bool = {|x| if $x { 'green' } else { 'light_red' } }
```